src/EnergyPlus/PlantChillers.cc

```diff
@@ -1259,7 +1262,13 @@ namespace PlantChillers {
 					ErrorsFound = true;
 				}
 				TestCompSet( cCurrentModuleObject, cAlphaArgs( 1 ), cAlphaArgs( 13 ), cAlphaArgs( 14 ), "Heat Recovery Nodes" );
-				RegisterPlantCompDesignFlow( EngineDrivenChiller( ChillerNum ).HeatRecInletNodeNum, EngineDrivenChiller( ChillerNum ).DesignHeatRecVolFlowRate );
+				if ( EngineDrivenChiller( ChillerNum ).DesignHeatRecVolFlowRate == DataSizing::AutoSize ) {
+					EngineDrivenChiller( ChillerNum ).DesignHeatRecVolFlowRateWasAutoSized = true;
+				} else {
+					RegisterPlantCompDesignFlow( EngineDrivenChiller( ChillerNum ).HeatRecInletNodeNum, EngineDrivenChiller( ChillerNum ).DesignHeatRecVolFlowRate );
```

Comment:
Presumably, if this field is autosized, you have to make this call later after sizing it.  If so, could we remove the call here for hard-sized flow registration?  That way it's only in one spot?